conans/cli/api/subapi/remotes.py

```diff
@@ -31,9 +31,12 @@ def list(self, pattern=None, only_active=False):
         return remotes
 
     @api_method
-    def get(self, remote_name):
+    def get(self, remote_names=None):
```

Comment:
I think a `get` is for 1 element. You can perfectly iterate to get N names calling N times.